src/signet.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2019-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <signet.h>
+
+#include <consensus/merkle.h>
+#include <consensus/params.h>
+#include <consensus/validation.h>
+#include <hash.h>
+#include <primitives/block.h>
+#include <primitives/transaction.h>
+#include <script/interpreter.h>
+#include <script/standard.h>        // MANDATORY_SCRIPT_VERIFY_FLAGS
+#include <util/system.h>
+
+int GetWitnessCommitmentIndex(const CBlock& block);
+template<typename T> int GetWitnessCommitmentIndex(const T& tx);
```

Comment:
Same for those two as well, obviously ;)

Reply:
Oh... yeah, okay. I moved the declarations as well (from `src/validation.h`).